components/FriendCodesPanel.tsx

```diff
@@ -39,16 +42,16 @@ function CopyButton({ copyText, copiedText, onClick }) {
 function FriendInviteCard({ invite }: { invite: FriendInvite }) {
     return (
         <div className="vc-friend-codes-card">
-            <Flex justify={Flex.Justify.START}>
+            <Flex justifyContent="start">
```

Comment:
Please update this to `space-between`
```suggestion
            <Flex justifyContent="space-between">
```